src/jquery.multiselect.js

```diff
@@ -43,60 +47,66 @@
-      iconSet: null,                       // (plain object | null) Supply an object of icons to use alternative icon sets, or null for default set.  Reference defaultIcons above for object structure.
-      checkAllText: 'Check all',           // (str | blank | null) If blank, only icon shown.  If null, no icon, text or link is shown.
-      uncheckAllText: 'Uncheck all',       // (str | blank | null) If blank, only icon shown.  If null, no icon, text or link is shown.
-      flipAllText: null, //'Flip all',     // (str | blank | null) If blank, only icon shown.  If null, no icon, text or link is shown.
-      noneSelectedText: 'Select options',  // (str) The text to show in the button where nothing is selected.
-      selectedText: '# of # selected',     // (str) A "template" that indicates how to show the count of selections in the button.  The "#'s" are replaced by the selection count & option count.
-      selectedList: 0,                     // (int) The actual list selections will be shown in the button when the count of selections is <= than this number.
-      selectedMax: null,                   // (int | function)  If selected count > selectedMax or if function returns 1, then message is displayed, and new selection is undone.
-      show: null,                          // (array) An array containing menu opening effects.
-      hide: null,                          // (array) An array containing menu closing effects.
-      autoOpen: false,                     // (true | false) If true, then the menu will be opening immediately after initialization.
-      position: {},                        // (object) A jQuery UI position object that constrains how the pop-up menu is positioned.
-      appendTo: null,                      // (jQuery | DOM element | selector str)  If provided, this specifies what element to append the widget to in the DOM.
-      menuWidth:null,                      // (int | null) If a number is provided, sets the menu width.
-      selectedListSeparator: ', ',         // (str) This allows customization of the list separator.  Use ',<br/>' to make the button grow vertically showing 1 selection per line.
-      htmlButtonText: false,               // (true | false) If true, then the text used for the button's label is treated as html rather than plain text.
-      htmlOptionText: false,               // (true | false) If true, then the text for option label is treated as html rather than plain text.
-      disableInputsOnToggle: true,         // (true | false)
-      groupColumns: false                  // (true | false)
+   $.widget("ech.multiselect", {
+
+   // default options
+   options: {
+      header: true,                       // (true | false) If true, the header is shown.
+      height: 175,                        // (int | 'size') Sets the height of the menu in pixels.  If 'size' is instead specified, the native select's size attribute is instead for height.
```

Comment:
Your comment here is kind of confusing.

Reply:
The 'size' setting just makes the height of the multiselect accommodate what is needed for the # of rows specified via the size setting of the native select.   This is actually very useful... it is something I use a lot w/ my implemented version of my old fork in my web app.

I am totally open to less confusing wording.

Reply:
It's mostly a grammar thing:
> the native select's size attribute is instead for height.
could maybe be
> the native select's size in rows attribute is used instead for height